demystifying_command_line/demystifying_command_line.md

```diff
@@ -0,0 +1,251 @@
+
+Learning how to quit out of things that are happening in the command line is a very common issue for brand new users, and it's something of a rite of passage.  
+
+There's a text editor that some people use from the command line called `vim` which is notoriously tricky to learn (we don't recommend it, we prefer `nano`).  So many people ask for help getting out of `vim` that ["How do I exit Vim?"](https://stackoverflow.com/questions/11828270/how-do-i-exit-vim) is one of [Stack Overflow's](https://www.stackoverflow.com) most frequently viewed posts! It's a question so popular that it [has its own memes](https://stackoverflow.blog/2017/05/23/stack-overflow-helping-one-million-developers-exit-vim/).  
+
+</div>
+
+And finally, let's talk about getting things wrong.  What will happen if you type the wrong thing?  Probably, **nothing** will happen.  The computer will give the equivalent of a shrug and say "I don't understand."  If you type a command that doesn't exist, or do something wrong, like put some words in the wrong order, you'll get an error message that might feel difficult to parse, but doesn't itself mean that anything got harmed.  
+
+Of course, it makes sense to check what a command actually does before typing it.  For example, the command `rm` removes something -- a file, or even a directory and everything in it.  If someone wanted to do a very mean prank they might suggest that you use a command including `rm`.  So after you search online but **before** you type a command in, do a bit of searching about what the command does.  
+
+
+## The Practical Use of GUIs and CLIs 
+
+Most things you want to accomplish on your computer can be done using multiple interfaces, allowing you to choose which you prefer -- graphical or command line. However, as you get into very specific or customized tasks that are done more rarely, there **are some things that can only be done by command line**.  It might be helpful to get a little comfortable with the command line interface on your computer now, before you discover that you have to use it for a specialized task.  If that's something that interests you, our [Bash / Command Line 101](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/bash_command_line_101/bash_command_line_101.md) module might be helpful!
+
+![A one-panel cartoon. A person with a ponytail is typing on a computer and says "Whoa, this is running MS-DOS!  It's weird how new technology takes forever to reach some industries".  Someone standing nearby replies, "Yeah.  Like how we still use gunpowder for fireworks, even though we've had nuclear weapons for over 70 years."](https://imgs.xkcd.com/comics/obsolete_technology.png "[Obsolete Technology](https://xkcd.com/1891/) comic by xkcd, [CC BY-NC 2.5](https://xkcd.com/license.html).")
```

Comment:
```suggestion
```

Consider removing this comic -- its not clear to me how it connects to the content on the page, other than the fact that MS-DOS lacked a modern GUI. Also, some learners (I'm mostly thinking about younger ones) may not be familiar with MS-DOS, and it doesn't seem valuable to throw extra technical terms at them if its not directly connected to the content.